p2p/src/rpc.rs

```diff
@@ -65,11 +61,6 @@ impl P2pRpcServer for super::P2pHandle {
         handle_error(res)
     }
 
-    async fn get_peer_id(&self) -> rpc::Result<String> {
```

Comment:
We still need the get_peer_id function. The peer id is a unique identifier for the node for operational purposes (like in functional tests), not for consensus purposes.

Reply:
I'm not sure it's a good idea to add such RPC call just for operation purposes. Bitcoin core manages without it.
Why RPC socket address or PID is not enough to identify the node you are talking to?

Reply:
That's actually not true. The peer id is used in bitcoin as a standard way to identify connections. See here as an example: 

https://github.com/bitcoin/bitcoin/blob/d8bdee0fc889def7c5f5076da13db4fce0a3728a/src/rpc/net.cpp#L406

Because we need a deterministic way to identify connections by peer info in scripting and similar.

Reply:
Right, I'm adding it now.
But I was referring to `get_peer_id` RPC call specifically, which is supposed to return some kind of "my node id", which is become meaningless now.